pkgs/tools/games/gamemode/default.nix

```diff
@@ -66,24 +58,20 @@ stdenv.mkDerivation rec {
   ];
 
   mesonFlags = [
-    # libexec is just a way to package binaries without including them
-    # in PATH. It doesn't make sense to install them to $lib
-    # (the default behaviour in the meson hook).
-    "--libexecdir=${placeholder "out"}/libexec"
-
+    "-Dwith-pam-limits-dir=etc/security/limits.d"
     "-Dwith-systemd-user-unit-dir=lib/systemd/user"
+    "-Dwith-systemd-group-dir=lib/sysusers.d"
+
+    # Meson installs internal executables to $lib/lib by default, but
```

Comment:
Did you mean this?
```suggestion
    # Meson installs internal executables to $out/lib by default, but
```

Reply:
Yep! When `"lib"` is in `outputs`, the meson builder by default will install these to the `$lib` output, but they should be installed under `$out/libexec`.

I'll update this comment to make it clearer.